src/blocks/homepage-articles/view.scss

```
@@ -344,7 +344,7 @@
 			right: 0;
 			text-align: right;
 			text-overflow: ellipsis;
-			z-index: 2;
+			z-index: 3;
```

```suggestion
			z-index: 3;
			
			a,
			a:visited {
			    color: #fff;
			}
```

Just need to add a little tweak for links in the caption to make sure they're not picking up the theme colour -- I tested this and it should work for regular & visited links in credits!

(I only think this is an issue because the theme's link styles are pretty specific!)

---

Thanks—totally forgot to test the different link states!